src/modules/jcms_rest/src/Plugin/rest/resource/AbstractRestResourceBase.php

```diff
@@ -359,9 +360,14 @@ protected function filterPageAndOrder(QueryInterface &$query, $sort_by = NULL) {
    * Apply filter by show parameter: all, open or closed.
    *
    * @param \Drupal\Core\Entity\Query\QueryInterface $query
-   * @param string
+   * @param string $filterFieldName The name of the field to filter on
+   * @param bool $isTimeStamp whether $field is a Timestamp field
+   * @throws InvalidArgumentException if the filter field name argument is not supplied
    */
-  protected function filterShow(QueryInterface &$query) {
+  protected function filterShow(QueryInterface &$query, string $filterFieldName = '', bool $isTimeStamp = FALSE) {
```

Comment:
If a value is expected for `$filterFieldName` then we should not set a default value of `''`.

Suggested change:
```
protected function filterShow(QueryInterface &$query, string $filterFieldName, bool $isTimeStamp = FALSE) {
```